packages/cli/README.md

```diff
@@ -20,3 +20,30 @@ Runs compilation in the current directory and all of its subdirectories.
 ### `npx squiggle-cli-experimental watch`
 
 Watches `.squiggleU` files in the current directory (and subdirectories) and rebuilds them when they are saved. Note that this will _not_ rebuild files when their dependencies are changed, just when they are changed directly.
+
+## Further instructions
+
+The above requires having node, npm and npx. To install the first two, see [here](https://nodejs.org/en/), to install npx, run:
+
+```
+npm install -g npx
```

Comment:
`npx` has been pre-bundled with `npm` [since 2017](https://github.com/npm/npm/releases/tag/v5.2.0), so most users will have it installed.